### PR TITLE
Channel sort QA

### PR DESCRIPTION
**Pull request by broskoski**
Closes some items from https://github.com/aredotna/ervell/issues/672

### CODE REVIEW

File: components/add_block/stylesheets/grid.styl [hunk 1]
```diff
@@ -81,6 +81,10 @@
     pointer-events none
 
   &__button
+    &:disabled
+      pointer-events all
```
Comment (dzucconi) [inline components/add_block/stylesheets/grid.styl]: Is this clickable then?

Comment (dzucconi) [inline components/add_block/stylesheets/grid.styl]: Because it should not be

Comment (dzucconi) [inline components/add_block/stylesheets/grid.styl]: And it shouldn't be hoverable anyway when it's disabled

Comment (broskoski) [inline components/add_block/stylesheets/grid.styl]: True, just a sec

Comment (broskoski) [inline components/add_block/stylesheets/grid.styl]: Confirmed that this rule is still applied after the user has typed

**Pull request merged**